test/hotspot/jtreg/compiler/vectorization/TestFloat16VectorSum.java

```diff
@@ -33,7 +33,7 @@
 package compiler.vectorization;
```

Comment:
I am not able to add comments to the lines above this line. I see "-XDenablePrimitiveClasses" flag on line #29 which I think is obsolete after the latest merge. If this test passes for you then you can leave this unchanged. I can change this line in my next PR. Thanks.